src/explore-education-statistics-common/src/components/AccordionSection.module.scss

```diff
@@ -1,29 +0,0 @@
-@mixin accordion-section-expanded() {
-  display: block !important;
-  height: auto;
-  padding: 15px 0 !important;
-  visibility: visible;
-}
-
-.section {
-  @media print {
```

Comment:
Have you checked how this looks in print mode? Looks like the point of this styling was to avoid page breaks occurring inside of the accordion sections

Reply:
I checked that the accordions were expanded in print mode, but not the page breaks - I'll check and readd it if necessary